ignite/pkg/chaincmd/in-place-testnet.go

```diff
@@ -0,0 +1,27 @@
+package chaincmd
```

Comment:
IMHO, we can add the method `func (c ChainCmd) TestnetInPlaceCommand(newChainID, newOperatorAddress string, options ...InPlaceOption) step.Option` in this file